lib/player.rb

```diff
@@ -0,0 +1,9 @@
+class Player
+  attr_reader :name
+  attr_accessor :choice
```

Comment:
Appropriate use of `reader` and `accessor`, good that you didn't make them both `attr_accessor`